encrypted-media-respec.html

```diff
@@ -2024,9 +2024,21 @@
 
     <section>
       <h2><a>HTMLMediaElement</a> Extensions</h2>
+      
+      <!-- Put all the monkey-patching here -->
+      
       <p>This section specifies additions to and modifications of the <a def-id="htmlmediaelement"></a> [[!HTML5]] when the Encrypted Media Extensions are supported.</p>
+      
+      <p>This specification adds two internal values to the <a def-id="htmlmediaelement"></a>: <var>waiting for key</var> which shall have a boolean value and <var>encrypted block queue</var> which shall be a queue of encrypted blocks awaiting decryption.</p>
 
-      <p>When a <a def-id="htmlmediaelement"></a> is created, its internal <var>waiting for key</var> value is initialized to <code>false</code>.</p>
+      <p>When a <a def-id="htmlmediaelement"></a> is created, its internal <var>waiting for key</var> value shall initialized to <code>false</code> and its <var>encrypted block queue</var> value shall be empty.</p>
```

Comment:
"<var>waiting for key</var> value shall *be* initialized"

i.e. insert "be" between "shall" and "initialized".